src/modules/musig/main_impl.h

```diff
@@ -502,6 +502,21 @@ int secp256k1_musig_partial_sig_combine(const secp256k1_context* ctx, const secp
         secp256k1_scalar_add(&s, &s, &term);
     }
 
+    /* If there is a tweak then add `msghash` times `tweak` to `s`.*/
+    if (tweak32 != NULL) {
+        unsigned char msghash[32];
+        secp256k1_scalar e, scalar_tweak;
+        int overflow = 0;
+
+        if (!secp256k1_musig_compute_messagehash(ctx, msghash, session)) {
+            return 0;
+        }
+        secp256k1_scalar_set_b32(&e, msghash, NULL);
+        secp256k1_scalar_set_b32(&scalar_tweak, tweak32, NULL);
```

Comment:
Hm, `ec_pubkey_tweak_add` checks for overflow... Maybe we should do the same just for consistency. (Not that it will matter...)

Reply:
I did this first and then removed it because of the additional complexity and the additional test for no good reason because `ec_pubkey_tweak_add` will already fail with the same tweak. However it's possible to trigger this condition if the tweak is not output of a hash function. Perhaps I should just add the check.

Reply:
done